doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,301 @@
+
+Results and discussion
+-----------------------
+
+The following average pressure and velocity fields are obtained for an angle of attack :math:`\alpha`such that :math:`\alpha \in \{0,5,7,9,11,15\}` : 
+
+.. image:: image/pressionMoy.png
+
+.. image:: image/vitesseMoy.png
+
+
+It is already noticeable that the the bigger the angle of attack, the bigger the pressure gradient. Following this observation, the lift coefficient :math:`C_L` is expected to increase with the angle of attack. The variation of the lift and drag coefficients are given below with a comparison to the litterature (Kouser et al. (2021) `[1] <https://doi.org/10.1177/17568293211055656>`_), they are computed using the following formula : 
+
+.. math::
+        C_L = \frac{F_L}{0.5\rho u^2 S} \; \; \; \; \; C_D = \frac{F_D}{0.5\rho u^2 S}
+        
+with :math:`F_L` and :math:`F_D` respectively the lift and drag forces. Those forces can be obtained in the ``force.00.dat`` and post processed using the ``postProcessing.py`` python file included in the folder of this example. :math:`S` represents a reference area, here it is equal to the product of the chord length :math:`C` (equal to 1 in this example) multiplied by a unitary transversal length.
+        
+.. image:: image/ClCdResultsPlot.png
+
+There is a slight variation of the results obtained compared to the litterature results. Indeed, the simulation is purely two-dimensionnal while Kouser et al. take a 2D slice of a 3D simulation. Still, the evolution observed is relevant.
+
+One can also see the low velocity zones on the upper part of the airfoil which corresponds to the boundary layer : the ``noslip`` condition on the NACA imposes a zero velocity condition on the fluid. This create a zone where the advective and diffusive forces are in competition. The following streamline representation helps to see the movements of the fluid inside the boundary layer : 
+
+.. image:: image/nacaStreamlineAngles.png
+
+It can be observed that zones of recirculation form on the airfoil. This is due to two phenomenas : first the flow outside of the boundary layer tends to "pull" it in its direction and the ``noslip`` boundary condition slows the fluid, then a positive pressure gradient -commonly refered to as adverse pressure gradient- on the airfoil surface pushes the fluid backwards. Following this a vertex forms. Below is represented the mean pressure coefficient :math:`C_p` on the airfoil with comparison to the litterature right below. It is computed using the following formula : 
+
+.. math::
+        C_p = \frac{p-p_{\infty}}{0.5\rho_{\infty}u_{\infty}^2}
+        
+with :math:`p_{\infty}` the static pressure in the freestream (equal to 0 in this case), :math:`\rho_{\infty}` the freestream fluid density, equal to the fluid density since we are solving an incompressible flow and :math:`u_{\infty}` the freestream velocity of the fluid, equal to ``1.0`` in this case.
+
+.. image:: image/CpPlot.png
+
+.. image:: image/cpPlotRef.png
```

Comment:
You can use Webplot Digitizer to extract the data of the reference graph and compare the results in a single figure :)

Reply:
Also, you can add the reference of the literature

Reply:
Reference added! 
 I thought it would be more readable to have two separate pressure coefficient plots since only the general for of it mattered to me